pkgs/development/python-modules/pytest-check/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-check";
+  version = "0.3.5";
+
+  src = fetchFromGitHub {
+    owner = "okken";
+    repo = "pytest-check";
+    rev = version;
+    sha256 = "11wb4f4sp4cr5mzqdakrbycwgfr2p1sx1l91fa6525wnfvgc0qy3";
+  };
+
+  checkInputs = [ pytest ];
+
+  propagatedBuildInputs = [
+    pytest
```

Comment:
We're preferring having `pytest` in `buildInputs` to make using multiple versions of pytest more convenient.

Reply:
Can we also use `pytestCheckHook` in `checkInputs` to eliminate overriding the `checkPhase`?

Reply:
Updated the PR. Didn't know about `pytestCheckHook`, thanks!